### ISSUE TITLE
Comparison with existing packages

### ISSUE THREAD

**Issue by bbatsov** [ema2159/centaur-tabs#25]
Hey there!

I came here to learn a bit about the package after seeing the PR to theme it in Zenburn and it looks pretty cool. One thing that's not clear to me, however, is what are the differences between `tabbar`, `awesome-tabs` and `centaur-tabs`, so you might want to add something on the subject to the README. You mentioned that the package started life as a fork of `awesome-tags`, but I'm curious to hear what was the motivation for the fork and so on.

**Comment by ema2159** [comment]
Hi mr. Batsov. First of all, thanks for all your work, you're goddamn awesome! I came from Atom and VS Code (and VIM also), and I came to notice that Emacs doesn't ships with tabs by default (and that for some reason if you mention the word "tabs" in front of a conservative Emacs user, they take it personal, like REALLY personal), but it is a component that really helps my workflow so I searched for a package to help me with that.
First, there  is tabbar (which may be considered an abandoned project), which I found very lacking. There was also tabbar-ruler, which I was interested in becoming a maintainer of but I think it's too bloated and the code is really messy (I have pretty high documentation and coding standards).
Then I found awesome tab (which the author doesn't say in the README but it is actually based in tabbar) and I was interested in contributing to the package, but the maintainer let me know that he wasn't willing to put the package in MELPA, so I asked if I could fork it and base my plugin in his plugin and he told me to feel free to do it.
Therefore, this package can be considered the successor of those plugins. Integrating a lot of their functionalities and improving them too. I was concerned on making it as eye pleasing and out of the box as possible so I took the task to send a lot of PRs to theme packages (which @ianpan870102 has helped me). There's a lot of work to do but overall people have really liked the package and I've been receiving a lot of positive feedback.
I hope I answered your questions!
